runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaTask.kt

```diff
@@ -6,10 +6,10 @@ import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.Nested
 import org.jetbrains.dokka.*
 
-abstract class DokkaTask : AbstractDokkaTask() {
+abstract class DokkaTask : AbstractDokkaLeafTask() {
 
     @get:Internal
-    val dokkaSourceSets: NamedDomainObjectContainer<GradleDokkaSourceSetBuilder> =
+    override val dokkaSourceSets: NamedDomainObjectContainer<GradleDokkaSourceSetBuilder> =
```

Comment:
Could you extract the implementation of this property and the second one in the `DokkaTaskPartial` class to the `AbstractDokkaLeafTask` instead of overriding it please, since the implementations are the same AFAIK